app/src/main/java/com/sunlightlabs/android/congress/utils/Utils.java

```diff
@@ -42,6 +43,10 @@ public static String decodeHTML(String input) {
         return input.replace("&#39;", "'");
     }
 
+	public static boolean checkPermission(Activity activity, String permission) {
+		return (ContextCompat.checkSelfPermission(activity, permission) == PackageManager.PERMISSION_GRANTED);
+	}
```

Comment:
This also checks for the call permission at run-time, and if we don't have it, then it defaults to loading the dialer with the phone number preloaded for the user to confirm (which doesn't require any permissions).